src/fmeval/reporting/constants.py

```diff
@@ -167,7 +167,8 @@ class ListType(str, Enum):
 # Score descriptions for the report
 SCORE_DESCRIPTIONS = {
     # Factual knowledge
-    FACTUAL_KNOWLEDGE: "A factual knowledge score is a value between 0 and 1, indicating the percentage of correctly retrieved real-world facts across different knowledge categories.",
+    EXACT_INCLUSION: "An exact inclusion score is a binary score where 1 indicates that the target output is contained in the model output, and 0 indicates otherwise.",
+    QUASI_EXACT_INCLUSION: "A quasi-exact inclusion is a binary score that represents whether a slightly modified version of the target output is contained in the model output. If, the score would be 1. Otherwise it would be 0.",
```

Comment:
Looking at the old description ("[...] indicating the percentage of correctly retrieved real-world facts across different knowledge categories."), this should be an explanation of the scores _aggregated over the whole dataset by taking the mean_. Thus, update to

 EXACT_INCLUSION: "Exact inclusion score is a binary score where 1 indicates that the target output is contained in the model output, and 0 indicates otherwise. The average over the whole dataset is reported, such that the final score indicates the percentage of correctly retrieved real-world facts.",
    QUASI_EXACT_INCLUSION: "Quasi-exact inclusion is a binary score that represents whether a slightly modified (removing excess whitespace and punctuation) version of the target output is contained in the model output. If yes, the score is 1, otherwise 0. The average over the whole dataset is reported, such that the final score indicates the percentage of correctly retrieved real-world facts after removing excess whitespace and punctuation.",